openshift/4.0/templates/logging/deploy-elasticsearch.yaml

```diff
@@ -1,227 +0,0 @@
-kind: Template
```

Comment:
If we're removing eleasticsearch then the API needs to be updated, as it is currently attempting to write to elasticsearch.